bin/acli

```diff
@@ -53,29 +57,13 @@ $output = new ConsoleOutput();
 $logger = new ConsoleLogger($output);
 $container = new Symfony\Component\DependencyInjection\ContainerBuilder();
 $container->setParameter('repo_root', find_repo_root());
-$container->set('amplitude', Amplitude::getInstance());
```

Comment:
@danepowell There was too much duplication here and in the TestBase class, so I've moved much of this to a static method that can be shared.